cmd/entrypoint-wrapper/main_test.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/openshift/ci-tools/pkg/api"
+)
+
+func TestManageCLI(t *testing.T) {
+	for _, testCase := range []struct {
+		testName string
+		cliDir   string
+		expected string
+	}{
```

Comment:
I would separate the test cases and the run of the test, like we do in other tests, take a look here for example: https://github.com/openshift/ci-tools/blob/master/cmd/github-ldap-user-group-creator/main_test.go#L33-L42